parquet-common/src/main/java/org/apache/parquet/VersionParser.java

```diff
@@ -31,7 +31,7 @@
  */
 public class VersionParser {
   // example: parquet-mr version 1.8.0rc2-SNAPSHOT (build ddb469afac70404ea63b72ed2f07a911a8592ff7)
-  public static final String FORMAT = "(.+) version ((.*) )?\\(build ?(.*)\\)";
+  public static final String FORMAT = "(.+) version( |$)?(.*?)( |$)?(\\(build ?(.*)\\))?";
```

Comment:
This re still seems a bit brittle to me. What about matching white-space with \s*?